sema/commons/store/store.py

```
@@ -353,16 +359,18 @@ def store_constr_rw():
 
     @property
     def sparql_store(self):  # dynamically (delayed) build of the instance
+        if self._store_constr is None:
+            raise RuntimeError("No store available")
```

Ruff

Avoid specifying long messages outside the exception class

(TRY003)